base/ca/src/main/java/com/netscape/cms/servlet/csadmin/GetCertChain.java

```diff
@@ -111,19 +111,17 @@ protected void process(CMSRequest cmsReq) throws EBaseException {
         chainBase64 = normalizeCertStr(chainBase64);
 
         try {
-            XMLObject xmlObj = null;
+            JSONObject jsonObj = new JSONObject();
 
-            xmlObj = new XMLObject();
+            ObjectNode responseNode = jsonObj.getMapper().createObjectNode();
+            responseNode.put("Status", SUCCESS);
+            responseNode.put("ChainBase64", chainBase64);
 
-            Node root = xmlObj.createRoot("XMLResponse");
+            jsonObj.getRootNode().set("Response", responseNode);
```

Comment:
We probably could store the `Status` and `ChainBase64` fields directly in the root node, but it's no big deal, the API will always evolve in the future versions.

Reply:
I did it like this to make the JSON response look the same as the XML response; principle of least astonishment. The XML root tag was called Response, this was the simplest way to make the JSON root tag Response - otherwise it looks like:
```
{
  "State" : "1",
  "Type" : "CA",
  "Status" : "running",
  "Version" : "foo",
  "ProductVersion" : "bar"
}
```

Reply:
That actually should be fine. We've been converting classes (e.g. `UserData`) from JAXB to JSON and the old XML root element usually disappears like above (i.e. becoming an unnamed top-level map). No worries, this can be discussed in future API enhancements.